strawberryfields/apps/qchem/vibronic.py

```diff
@@ -207,15 +258,9 @@ def sample(
             for i in range(n_modes):
                 sf.ops.S2gate(t[i]) | (q[i], q[i + n_modes])
 
-        sf.ops.Interferometer(U1) | q[:n_modes]
-
-        for i in range(n_modes):
-            sf.ops.Sgate(r[i]) | q[i]
+        op = VibronicTransition(len(q))
```

Comment:
I think it's ok (since also tests pass), but I was wondering if it's bad practice to do this within the context statement.

Reply:
I brought `op = VibronicTransition(n_modes)` out of the context statement.